src/test/java/com/amazon/opendistroforelasticsearch/knn/KNNRestTestCase.java

```diff
@@ -543,6 +543,16 @@ protected void clearCache() throws Exception {
         updateClusterSettings(KNNSettings.KNN_CACHE_ITEM_EXPIRY_TIME_MINUTES, null);
     }
 
+    /**
+     * Clear script cache
+     *
+     * Remove k-NN script from script cache so that it has to be recompiled
+     */
+    protected void clearScriptCache() throws Exception {
+        updateClusterSettings("script.context.score.cache_expire", "0");
```

Comment:
Is this line required?

Reply:
Yes, this is what causes cache to evict the script. Here is documentation: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/modules-scripting-using.html#modules-scripting-using-caching

Reply:
Got it. Thanks